maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java

```diff
@@ -63,32 +62,29 @@ protected void setUp()
             "src/test/resources/mojos/checkout/checkoutWithConnectionUrl.xml" ) );
         checkoutMojo.setWorkingDirectory( new File( getBasedir() ) );
 
-        String connectionUrl = checkoutMojo.getConnectionUrl();
-        connectionUrl = StringUtils.replace( connectionUrl, "${basedir}", getBasedir() );
-        connectionUrl = StringUtils.replace( connectionUrl, "\\", "/" );
-        checkoutMojo.setConnectionUrl( connectionUrl );
+        setupConnectionUrl( checkoutMojo );
 
         checkoutMojo.setCheckoutDirectory( checkoutDir );
 
         checkoutMojo.execute();
     }
 
+    private static void setupConnectionUrl( AbstractScmMojo mojo )
+    {
+        String connectionUrl = mojo.getConnectionUrl();
+        connectionUrl = connectionUrl.replace( "${basedir}", getBasedir() );
+        connectionUrl = connectionUrl.replace( '\\', '/' );
+        mojo.setConnectionUrl( connectionUrl );
+    }
+
     public void testTag()
         throws Exception
     {
-        if ( !ScmTestCase.isSystemCmd( SvnScmTestUtils.SVNADMIN_COMMAND_LINE ) )
```

Comment:
Why did you remove this test?

Reply:
It's not a test, it's an assumption and it's duplicative of the same code in the setUp method. If the code fails there, then the test isn't set up correctly here.

Reply:
Is the test really skipped w/o it?

Reply:
OK. I've added an explicit assumption in the setUp method to cover this.

Reply:
Jenkins passed